tests/kani/Intrinsics/Sysconf/main.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright Kani Contributors
```

Comment:
Can you please move this test to:
`tests/kani/LibC/sysconf.rs`

Reply:
Implemented as suggested.